src/components/logs/Lines.tsx

```diff
@@ -5,13 +5,19 @@ import { hasLength } from 'utils/misc-utils';
 import { useLogsContext } from './Context';
 import LogLine from './Line';
 import Spinner from './Spinner';
+import { SpinnerOptions } from './types';
 
 interface Props {
     height: number;
-    disableSpinner?: boolean;
+    spinnerOptions?: SpinnerOptions;
 }
 
-function LogLines({ height, disableSpinner }: Props) {
+function LogLines({ height, spinnerOptions }: Props) {
+    const disableSpinner = spinnerOptions?.disable ?? false;
+    const runningKey = spinnerOptions?.messages?.runningKey ?? undefined;
+    const stoppedKey = spinnerOptions?.messages?.stoppedKey ?? undefined;
+    const severity = spinnerOptions?.severity ?? undefined;
```

Comment:
Now we need to be able to handle more spinner options to allow for more detail in the logging.